code/datums/uplink_items/uplink_general.dm

```diff
@@ -215,6 +215,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	reference = "CH"
 	item = /obj/item/butcher_chainsaw
 	cost = 65
+	surplus = 0 // This has caused major problems with un-needed chainsaw massacres. Bwoink bait.
```

Comment:
I like a lot of the changes here, but I still think some of these items should be available, even if they're shit or overpowered. Imo, these should all be set to `surplus = 5`. For me, part of the whole thing about surpluses is that its a big risk for a big reward. I think while we should lower both the risk and reward, it shouldnt be all the way to zero
```suggestion
	surplus = 5 // Rarer because of un-needed chainsaw massacres. Bwoink bait.
```

Reply:
I can't say I particularly agree. There are still a number of very viable items that can be gotten from surplus crates, and in some instances are even much better now with a lot of the ammunition and poisons removed.